docs/docs/general/08-askui User Portal/token.md

```diff
@@ -15,10 +15,13 @@ With the workspace created you can now create access tokens for it. Click on the
 
 ![Overlap](./create_token_start.png)
 
-Enter the token name and an expiry date and click _create_.
+Enter the access token name and an expiry date and click _create_.
```

Comment:
I absolutely love how thorough you are.

Reply:
I mean like: What do you have a nice code editor search for? ;)